client/client.go

```diff
@@ -27,7 +27,7 @@ func PrepareRelease(release *deployer.Release, zip_file_path *string) error {
 		return err
 	}
 
-	lambda_arn := to.LambdaArn(release.AwsRegion, release.AwsAccountID, release.LambdaName)
+	lambda_arn := to.LambdaArn(release.AwsRegion, release.AwsAccountID, release.TaskResourceName)
```

Comment:
I think the rename of `LambdaName` to `ResourceName` is not necessary. As a step deployer, it will always be a lambda.